cli/README.md

```diff
@@ -0,0 +1,88 @@
+# Library for building CLI for Exonum node
+
+[![Travis Build Status](https://img.shields.io/travis/exonum/exonum/master.svg?label=Linux%20Build)](https://travis-ci.com/exonum/exonum)
+[![License: Apache-2.0](https://img.shields.io/github/license/exonum/exonum.svg)](https://github.com/exonum/exonum/blob/master/LICENSE)
+![rust 1.36.0+ required](https://img.shields.io/badge/rust-1.36.0+-blue.svg?label=Required%20Rust)
+
+`exonum-cli` crate provides an extensible command line interface for Exonum
+nodes.
+
+By default, the following CLI subcommands are available:
+
+| Command            | Description
+| ------------------ | -----------
+| generate-template  | Generate common part of the nodes configuration
+| generate-config    | Generate public and private configs of the node
+| finalize           | Generate final node configuration
+| run                | Run the node with provided node config
+| run-dev            | Run the node with auto-generated config
+| maintenance        | Perform different maintenance actions
+| help               | Prints this message or the help of the given subcommand(s)
```

Comment:
`Prints` of `Print` ? I'd prefer third form of verb here.

Reply:
This was copy-pasted from the actual `exonum-executable --help` output. I think it will be the best to leave things here as in output.